packages/api/src/controllers/baseController.js

```diff
@@ -195,7 +195,8 @@ export default {
     return await model
       .query(trx)
       .context({ user_id: req?.auth?.user_id, ...context })
-      .insertGraph(removeAdditionalPropertiesWithRelations(model, data));
+      .insertGraph(removeAdditionalPropertiesWithRelations(model, data))
+      .returning('*');
```

Comment:
I am seeing that internal_identifier is returned without this. So I am not sure I understand.

Reply:
Could you try sending a post request and checking what's returned?

Reply:
Yup my bad! I did on /animals but its only used for /animal_batches.

So only comment is that maybe the baseController function should be renamed to `insertGraphWithResponse()` to match the `postWithResponse()`.

Reply:
I will rename it, thank you!